.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+Please ensure this checklist is followed for any pull requests for this repo. This checklist must be checked by both the PR creator and by anyone who reviews the PR.
+* [ ] Relevant documentation for this PR has to be completed and reviewed by the documentation team (Paige, Marshall, or Eirik) before the PR can be merged
```

Comment:
```suggestion
* [ ] Relevant documentation for this PR has to be completed and reviewed by the documentation team (@ioptio, @mdr0id, or @Eirik0) before the PR can be merged
```

This would inform PR authors of the GitHub users they need to interact with regarding documentation. However, it would also I believe mean that these three get notified by every single PR that is opened. Not sure whether that is a bug or a feature :smile: